plugins/modules/pfsense_haproxy_frontend.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, Frederic Bor <frederic.bor@wanadoo.fr>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = """
+---
+module: pfsense_haproxy_frontend
+version_added: "2.10"
```

Comment:
This should be 0.6.0 at the moment.

Reply:
yep i made this over a year ago and never did a pr. client got back to me an dnow i'm having to get it to work again. i'll fix that.